### PR TITLE
Publish steps for docs website

### PR DESCRIPTION
**Pull request by benjaminpkane**
Adds the publishing steps via `gsutil` to automate the release of docs updates. @brimoor is there a cloud storage service account secret we can use? Or I can create one. cc @findtopher

### CODE REVIEW

**Review by brimoor — COMMENTED** [review]
@benjaminpkane is there risk that a docs build would fail in the action that would cause an incomplete/corrupted version of the docs to be synced to docs.voxel51.com? Or am I over-analyzing?

I recommend creating new GCS creds specifically for this workflow, just to be proper.

**Comment by benjaminpkane** [comment]
> @benjaminpkane is there risk that a docs build would fail in the action that would cause an incomplete/corrupted version of the docs to be synced to docs.voxel51.com? Or am I over-analyzing?

I would consider it unlikely, but if we do want to roll back a re-upload should take less than a minute

File: .github/workflows/build-docs.yml [hunk 1]
```diff
@@ -76,36 +76,16 @@ jobs:
...
-
-            voxel51/fiftyone@${{ github.sha }}
-          title: Automated fiftyone documentation update
-          body: " "
-          token: ${{ secrets.FIFTYONE_GITHUB_TOKEN }}
-          author: voxel51-bot <bot@voxel51.com>
-          labels: documentation
-          reviewers: brimoor, benjaminpkane
-          team-reviewers: developers
+          credentials_json: "${{ secrets.GCP_CREDENTIALS }}"
```
Comment (brimoor) [inline .github/workflows/build-docs.yml:87]: Needs updating to `DOCS_GCP_CREDENTIALS`?

**Review by brimoor — APPROVED** [review]
LGTM

**Pull request merged**